dashing/distribution.yaml

```diff
@@ -995,6 +995,24 @@ repositories:
       url: https://github.com/ros2/ros1_bridge.git
       version: master
     status: developed
+  ros2_intel_realsense:
+    doc:
+      type: git
+      url: https://github.com/intel/ros2_intel_realsense.git
+      version: 2.0.3
+    release:
+      packages:
+      - realsense_camera_msgs
+      - realsense_ros2_camera
+      tags:
+        release: release/dashing/{package}/{version}
+      url: https://github.com/ros2-gbp/ros2_intel_realsense-release.git
+      version: 2.0.3-1
+    source:
+      type: git
+      url: https://github.com/intel/ros2_intel_realsense.git
+      version: 2.0.3
```

Comment:
Please change this to a branch instead of the tag `2.0.3`. `version` must be a branch in the `source` block: https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#continuous-integration-indexing . I think this is why CI failed

```
ERROR: In file '''/home/travis/build/ros/rosdistro/dashing/distribution.yaml''': Could not validate source entry for repo ros2_intel_realsense with error [[[Could not validate repository with url https://github.com/intel/ros2_intel_realsense.git and version 2.0.3 from entry at line 1011. Error reason: Tags are not valid, but a tag 2.0.3 was found. Re: https://github.com/ros/rosdistro/pull/20286]]]
```